pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   googleapis_auth: ^0.2.11
   asn1lib: ^0.6.4
   pointycastle: ^1.0.2
+  openid_client: ^0.2.5
```

Comment:
This is unnecessary

Reply:
It is used to call .createCredentials

Reply:
Which is useless...

Reply:
How else would I make credentials for an auth token? I don't think firedart's implementation of auth does this anywhere.
Open id is the recommended way to do this: https://firebase.google.com/docs/reference/admin/node/admin.auth.DecodedIdToken

Reply:
You really should learn about how oauth works before writing code for it.
Token parsing is implemented in `jwt.dart`.

Reply:
Found it, thanks.